### ISSUE TITLE
[BUG]: RPT_Style_HinderFocus1 - triggering on non-focusable items

### ISSUE THREAD

**Issue by patsmith-public** [IBMa/equal-access#811]
### Project

engine

### Browser

Chrome, Safari, Firefox

### Description

Applies to DSC WFM draft comments. Code sandbox:
https://codesandbox.io/s/issue-811-9fz49g

Tom's comments:
* Note - initial issue noted color contrast, which wasn't exactly correct. I changed the description to better cover the actual issue being encountered. Ignore some of the comments below regarding color contrast.
* I don't think this should be triggering on something that's not focusable / tabbable
* Can we get the actual CSS color of the outline and check? 
* Since we can get computed values now, can we not get this to the point of only triggering on real violations? Feels like this may be closely related to the work Mike has been doing with WCAG an focus-visible.

**Comment by tombrunet** [comment]
Sorry - you're going to have to work with the DCS team on a solution to this. This isn't the right place to make those kinds of changes. Also, I do not have access to that report to see your issues.

Note that this is a general, open source tool for accessibility checking, and we are not part of the DCS team. DCS is simply using this engine. Generally, our team can review two types of issues:
1) Violations that shouldn't be violations (false positives / bugs). Many of the reports in this category are real violations, so we generally need a real example to confirm that it's a bug.
2) Needs Review items that we can make more specific. We have to make sure that this won't ignore significant errors.

We won't be able to support modifying rules to ignore real violations due to decisions of specific applications.

**Comment by nfernho** [comment]
@patsmith-public - Tom is correct. This is something that the DCS team can fix. Please open a DCS GitHub issue. Thank you.

**Comment by patsmith-public** [comment]
@nfernho I did https://github.ibm.com/ibm-product-documentation/requests/issues/565#issuecomment-42592415 

Please reopen the one that @hahnrob closed

**Comment by tombrunet** [comment]
FYI - doesn't seem this is about color contrast exactly. I'm trying to get a code snippet of a draft comment to get a better idea of what's really happening here.

**Comment by patsmith-public** [comment]
<div style="background-color: #99FF99; border: 1pt black solid;" class="draft-comment">

**Comment by tombrunet** [comment]
Okay, I'm going to change the title and description of this issue. It's something we may be able to tackle. It's about the outline and not the color. We may be able to tighten this up to only check on tabbable items. Or we can better check the outline color.

**Comment by shunguoy** [comment]
see the details in the description

**Comment by philljenkins** [comment]
no change to Help or Messages needed.
